### ISSUE TITLE
Suggestions for better user experience

### ISSUE THREAD

**Issue by Z-42PO** [Cobinja/CobiWindowList#4]
I was a user of Icing Task Manager, but for now, it's heavy and little bit buggy (CPU usage saturation). I don't know if CobiWindowList is still in development, but anyway, I use it now and here are some suggestions to (in my opinion) make it better.

1. add hotkey Super+1, Super+2, Super+3... to launch apps and switch between them with the keyboard
2. add rollover effect on the icons, no matter there are pinned or not
3. make some visual effect to make the difference between "launched opened-window app", "launched minimized-window app" and "not launched app" (get some inspiration from Win10 or "Dash To Panel" extension for gnome-shell)
4. add a setting to change the width of the tile, and the space between the tiles

Thx for this app, so far it's the only one working for me (Cinnamon 3.4.3 + Linux Mint 18.2 Sonya)

**Comment by Cobinja** [comment]
About those suggestions:
For 1. thinking about it
For 2. Rollover/Hover style is set by the Cinnamon theme. I will not implement an own transition effect
For 3. Also theme-based
For 4. The width depends on some of my settings ("Display caption for" and the label width) and the height of the panel, making icon-only tiles squared. It's by design.
Spacing depends on the used Cinnamon theme

**Comment by Z-42PO** [comment]
Yes I discovered it out recently. But what classes control those effects ? I've a right configured theme, working great with Icing Task Manager, but there is no visual effect with CobiWindow.
I think CobiWindow doesn't use the same classes...
Do you have any clue ?

**Comment by Cobinja** [comment]
I'm using the same classes as Cinnamon's builtin windowlist

- .window-list-item-box for the app buttons
- .window-list-box for the whole thing
- .top, .bottom, .left and .right depending on the panel location
- .window-list-item-demands-attention for app buttons that, well, demand attention

In general, just look for anything related to window-list-item in cinnamon.css
Additionally, look at my applet file stylesheet.css. In there, I have a :neutral pseudo class which is used while a pinned app button doesn't manage any window.

**Comment by Z-42PO** [comment]
Thanks, I'll try some tweaks one day

**Comment by Skokan03** [comment]
Hi,
I am trying to customize Linux Mint, and I am using your Window List.
But when I change panel height, underline (bottom border) disappears, other borders (left, top, right) stays.
On original window list, bottom border stays.
What am I doing wrong?
Thanks for answer and sorry for my English, I am just 14 years old kid playing with computer.
E: I am using Mint-Y and editing it.

**Comment by Cobinja** [comment]
@Skokan03 I can't reproduce this. To what size do you set the panel?
